training/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: CockroachDB Training
+summary: Learn how to use CockroachDB for your applications
+toc: false
+sidebar_data: sidebar-data-training.json
+---
+
+Welcome to the CockroachDB training.
+
+## Day 1
+
+### Overview
+
+- [Day 1 Overview](day-1-overview.html)
```

Comment:
Link should be to `day-1.html`. Though I'd suggest we just rename the file to match this link, as it's more descriptive and aligns with the recap file: `day-1-overview.md`